config/config.go

```diff
@@ -103,7 +104,10 @@ func (c configErrors) Error() string {
 func (cfg *Configuration) validate() configErrors {
 	var errs configErrors
 	errs = cfg.AuctionTimeouts.validate(errs)
-	errs = cfg.StoredRequests.validate(errs)
+	errs = cfg.StoredRequests.validate("stored_req", errs)
```

Comment:
This PR is build on top of https://github.com/prebid/prebid-server/pull/1446. In that PR, I recommend moving where `DataType` lives. Applying those comments to this PR, I think `validate` would be a good spot to pass down the data types.

Due to the dependency, let's try to resolve the other PR first.